iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ThreeWindingsTransformerImpl.java

```diff
@@ -359,6 +359,12 @@ public List<Leg> getLegs() {
         return Arrays.asList(leg1, leg2, leg3);
     }
 
+    @Override
+    public ThreeWindingsTransformer setRatedU0(double ratedU0) {
+        this.ratedU0 = ratedU0;
```

Comment:
add validation and notification of change, similar to `setRatedU` of each leg

Reply:
Done